pyproject.toml

```diff
@@ -100,10 +100,6 @@ filterwarnings = [
     '''default:ssl NPN is deprecated, use ALPN instead:DeprecationWarning''',
 ]
 
-[tool.isort]
-profile = "black"
-add_imports = "from __future__ import annotations"
```

Comment:
This is the one line that I could not figure out how to replicate with ruff.
* https://beta.ruff.rs/docs/settings/#isort

Reply:
https://github.com/charliermarsh/ruff/pull/1762

Reply:
Thanks!! Fixed.